sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -802,7 +808,7 @@ intervalUnit
     ;
 
 colPosition
-    : FIRST | AFTER qualifiedName
```

Comment:
can we remove the definition of `qualifiedName` if it's used nowhere?

Reply:
in current change i just replaced qualifiedName for DDL commands.

there are still few places using qualifiedName. Need to address them see if we can replace them too.

Reply:
Can a function be named as test-func without quote?

There are rules `CRATE (OR REPLACE)? TEMPORARY? FUNCTION...`  and `DROP TEMPORARY? FUNCTION` that use `qualifiedName`.

Reply:
`qualifiedName` can't match `a-b`. I take another look, `qualifiedName` is still needed in places where `-` is a legal following, e.g. `select a - b`, we must use `qualifiedName` to match column name.

For function name, we can use `multipartIdentifier`

Reply:
If we use `multipartIdentifier` in CREATE FUNCTION, meaning function name is `multipartIdentifier`.  Will it be a problem for parsing `SELECT a-test_func(...) ...`?

In this case, a-test will be parsed by `multipartIdentifier` and an exception will be thrown to ask adding quote for it.

Reply:
We can only use `multipartIdentifier` when define the function, e.g. `CREATE FUNCTION ...`, not when calling the function.